_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
+- Metadata
+- Configuration
+- Security
+- Networking
+- Storage
+- Cluster Operation
+- Workloads
+- Architecture
```

Comment:
@heidecke Preferably as a way of easily identifying the sub-components/processes that are essential to Kubernetes functionality (but aren't necessarily things you need to pay attention to all the time)

e.g. etcd, apiserver, node controller, kubelet

Let me know if that doesn't make sense!